beekeeper-code-analysis-plugin/src/main/java/io/beekeeper/gradle/code/CodeAnalysisPlugin.java

```diff
@@ -54,8 +49,8 @@ private void configureTasks(Project project) {
                 task.setEnabled(false);
             } else {
                 boolean isJenkins = isJenkins();
-                task.getReports().getXml().setEnabled(isJenkins);
-                task.getReports().getHtml().setEnabled(!isJenkins);
+                task.getReports().maybeCreate("html").setEnabled(isJenkins);
```

Comment:
In the old version this was like this: 
```suggestion
                task.getReports().maybeCreate("html").setEnabled(!isJenkins);
```